subsys/bluetooth/host/att.c

```diff
@@ -497,8 +497,9 @@ static u8_t find_info_cb(const struct bt_gatt_attr *attr, void *user_data)
 		/* Fast forward to next item position */
 		data->info128 = net_buf_add(data->buf, sizeof(*data->info128));
 		data->info128->handle = sys_cpu_to_le16(attr->handle);
-		memcpy(data->info128->uuid, BT_UUID_128(attr->uuid)->val,
-		       sizeof(data->info128->uuid));
+		(void)memcpy(data->info128->uuid,
+				BT_UUID_128(attr->uuid)->val,
+				sizeof(data->info128->uuid));
```

Comment:
It doesn't look like the alignment is correct here for the opening parenthesis of the first line.